packages/www/src/components/Footer.js

```diff
@@ -0,0 +1,11 @@
+import { h } from 'preact'
+
+const Footer = () => {
+  return (
+    <div className='pb-3 px-5 text-center'>
+      <span> Last updated {new Date().getFullYear()}</span>
```

Comment:
I don't think this serves us well. We are implying that the date here is the date the site's code or content was updated, but with a dynamic date that might not be true.

If you want to scaffold out a footer, we could put the Twitter link as some temp content. Also fine to not have a footer until we have time to revisit the design and copy properly.

Thoughts?

Reply:
Def agree! I'll remove the component for now and we can always build something out when there are more routes to uh- navigate